src/logistration/LoginHelpLinks.jsx

```diff
@@ -44,10 +53,8 @@ const LoginHelpLinks = (props) => {
 
   const renderLoginHelp = () => (
     <div className="login-help">
-      { page === LOGIN_PAGE ? forgotPasswordLink() : signUpLink()}
-      <a className="field-link" href="https://support.edx.org/hc/en-us/sections/115004153367-Solve-a-Sign-in-Problem">
-        {intl.formatMessage(messages['logistration.other.sign.in.issues'])}
-      </a>
+      { page === LOGIN_PAGE ? forgotPasswordLink() : signUpLink() }
+      { loginIssueSupportURL() }
```

Comment:
You could pass `getConfig()` to the function to save an extra call.
```suggestion
      { loginIssueSupportURL(getConfig()) }
      
      loginIssueSupportURL = (config) => (config.LOGIN_ISSUE_SUPPORT_LINK
      ...
      )
```